views/tasks-widget.html

```diff
@@ -0,0 +1,21 @@
+<div class="widget" ng-controller="widgetController">
+
+    <form name="newTaskForm" novalidate ng-submit="addTask();">
+        <div class="new-task">
+            <textarea required ng-model='description' name="description" class="task-txt new-task-txt" rows=1
+                      placeholder="Enter your task description"></textarea>
+
+            <input ng-model='assignee' class='assign inline-edit' placeholder="Assign to">
```

Comment:
please implement some custom validator also
for example, check that user is one of the users you have in db (in array)